flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -330,6 +333,20 @@ public CompletableFuture<JobDetailsInfo> getJobDetails(JobID jobId) {
         return sendRequest(detailsHeaders, params);
     }
 
+    /**
+     * Requests the job exception history.
+     *
+     * @param jobID The job id
+     * @return Job exceptions
+     */
+    public CompletableFuture<JobExceptionsInfoWithHistory> getJobExceptions(JobID jobID) {
```

Comment:
I am curious what happens if there are loops / repeated exceptions - will this cause an issue with the rest call. Is there a case to have a paging API here to avoid large responses?